test/unit/lasparser.js

```diff
@@ -1,22 +1,19 @@
 import assert from 'assert';
-import { HttpsProxyAgent } from 'https-proxy-agent';
+import fs from 'fs';
 import LASParser from 'Parser/LASParser';
-import Fetcher from 'Provider/Fetcher';
 import { compareWithEpsilon } from './utils';
 
+const urlData = './test/data/pointclouds/';
+const lasFile = fs.readFileSync(`${urlData}/data_test.las`);
+const lazFile = fs.readFileSync(`${urlData}/data_test.laz`);
+const lazV14File = fs.readFileSync(`${urlData}/ellipsoid-1.4.laz`);
```

Comment:
Those two binary files have a total size of 619kB, I'll try to generate another dataset!